language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/BallerinaCodeActionProvider.java

```diff
@@ -0,0 +1,52 @@
+/*
+
```

Comment:
Additional new line.. Check all the files.

Reply:
Checked and removed additional new lines